web_src/less/features/heatmap.less

```diff
@@ -31,20 +28,38 @@
   }
 
   @media @mediaLgAndDown {
+    & {
+      min-height: 105px;
+    }
     .total-contributions {
       left: 21px;
     }
   }
 
   @media (max-width: 1000px) {
+    & {
+      min-height: 80px;
+    }
     .total-contributions {
       font-size: 10px;
       left: 17px;
-      bottom: -2px;
+      bottom: -4px;
```

Comment:
liked this better aligned with `inline`

Reply:
This is absolutely positioned, `display: inline` on the parent wouldn't do anything.

Reply:
> This is absolutely positioned, `display: inline` on the parent wouldn't do anything.

not here but on the `user-heatmap` div and I think it's more elegant that way.
then this would not be needed.
but if it works, it works.